src/helpers/misc.js

```diff
@@ -151,7 +151,7 @@ const solidityType = inputType => {
   if (inputType.includes('[]')) {
     return { type: 'string', solidityType: `${inputType}` };
   }
-  if (inputType.includes(uint)) return { type: 'number', solidityType: uint };
+  if (inputType.includes(uint)) return { type: 'string', solidityType: string };
```

Comment:
this should be fine as it was before ` if (inputType.includes(uint)) return { type: 'number', solidityType: uint };`